components/menu-dropdown/menu-dropdown.jsx

```diff
@@ -58,7 +58,7 @@ import { MENU_DROPDOWN, MENU_DROPDOWN_TRIGGER, LIST } from '../../utilities/cons
 // clicks even when those clicks are over areas that wouldn't normally fire
 // click or touch events (for example, iframes). A single overlay is shared
 // between all dropdowns in the app.
-const overlay = document.createElement('span');
+const overlay = typeof (document) !== 'undefined' ? document.createElement('span') : { style: {} };
```

Comment:
Seeing `typeof (document) !== 'undefined'` cached in a `const documentDefined` and used could be nice. Not a blocker at all.

Reply:
Good idea.